office-js/index.d.ts

```diff
@@ -108,9 +108,9 @@ declare namespace Office {
          */
         width?: number,
         /**
-         * Optional. Determines whether the dialog is safe to display within a Web frame.
+         * Optional. Determines whether the dialog box should be displayed within an IFrame. This setting is only applicable in Office Online clients, this setting is ignored by desktop clients. 
```

Comment:
grammatically, I think it would make more sense to day:

This setting is only applicable in Office Online clients, **and is ignored on the other platforms.**

But I'm also fine having this committed as is.